packages/library-base/src/commonMain/kotlin/io/realm/notifications/Cancellable.kt

```diff
@@ -13,12 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-package io.realm
+package io.realm.notifications
 
 /**
  * A _cancellable_ representing ongoing tasks or subscription that can be cancelled.
  */
-internal interface Cancellable {
-    fun cancel()
+public interface Cancellable {
```

Comment:
Same...the callback API's are not publically available yet.

Reply:
Made it internal and moved them into `io.realm.notifications.internal`